src/setupTests.js

```diff
@@ -0,0 +1,4 @@
+import { configure } from 'enzyme';
```

Comment:
This is probably standard; didn't know if this would pollute the global test namespace, or if we should rename it to something like `configure_enzyme`.

Reply:
Looked into the react-testing-library suggested by @JacobMGEvans and will remove this file entirely as it's not needed if we don't use Enzyme. While I like Enzyme the [react-testing-library](https://testing-library.com/docs/dom-testing-library/intro) provides a lot of great functionality and has support for vue, angular, preact, reason, and svelte should anyone decide to do something fun 👍